src/types.js.flow

```diff
@@ -124,7 +126,7 @@ export type Route = {
   /**
    * The resources for the route
    */
-  resources?: RouteResource[],
+  resources?: RouteResource<any>[],
```

Comment:
Would mixed be better here? Though I know flow handles it weirdly, so it might be painful to actually use

Reply:
Good callout! I did try to use `mixed` and `empty` here but unfortunately neither worked without causing further pain when integrating these changes into Jira's frontend. 😓  I suspect it'd need further work to make it work nicely.